### PR TITLE
Implement Stage 1 of cloud-init data generation

### PR DESCRIPTION
**Pull request by ivan4th**
This includes switching from flexvolume-based cloud-init specs to annotations.

---
This change is [Reviewable](https://reviewable.io/reviews/mirantis/virtlet/300)

### CODE REVIEW

**Comment by vefimova** [comment]
+1, after fixing disccussed nits with Piotr

File: docs/volumes.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ From [Libvirt spec](http://libvirt.org/formatdomain.html#elementsDisks):
 > **target**
 > The target element controls the bus / device under which the disk is exposed to the guest OS. The dev attribute indicates the "logical" device name. The actual device name specified is not guaranteed to map to the device name in the guest OS. Treat it as a device ordering hint
 
-4. Attached disks are seen inside OS as hard disk devices called dev/vdb, dev/vdc and so on. As said above there is no fixed behaviour form device names as well as regarding to the order on PCI bus.
+4. Attached disks are visible by the OS inside VM as hard disk devices `/dev/vdb`, `/dev/vdc` and so on. As said above there is no fixed behaviour form device names as well as regarding to the order on PCI bus.
```
Comment (jellonek) [inline docs/volumes.md]: for device names?

**Comment by vefimova** [comment]
:lgtm:

---

Reviewed 28 of 28 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/300#-:-Kl_8ntn3EpFPG_zLF1r:bnfp4nl)*

**Comment by jellonek** [comment]
:lgtm:

---

Reviewed 28 of 28 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[docs/volumes.md, line 45 at r1](https://reviewable.io:443/reviews/mirantis/virtlet/300#-Kl_9rpHRRQ1eaBnUNRS:-Kl_9rpIXRZtOELyBBcu:b-1fmi8m) ([raw file](https://github.com/mirantis/virtlet/blob/fa90d2f6f30ff23f5ff7877d17cb1a9c545d5df3/docs/volumes.md#L45)):*
> ```Markdown
> > The target element controls the bus / device under which the disk is exposed to the guest OS. The dev attribute indicates the "logical" device name. The actual device name specified is not guaranteed to map to the device name in the guest OS. Treat it as a device ordering hint
> 
> 4. Attached disks are visible by the OS inside VM as hard disk devices `/dev/vdb`, `/dev/vdc` and so on. As said above there is no fixed behaviour form device names as well as regarding to the order on PCI bus.
> ```

It's not introduced there, so it's not blocking this PR, but i'm noting there that i don't quite understand second sentence in this point. Probably there should be `s/form/for/`.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/300#-:-Kl_AKbaHXCXANnX5GaP:bnfp4nl)*

**Comment by jellonek** [comment]
:lgtm:

---

Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/300#-:-KlbhwpjSCBeHxXJKS_Q:bnfp4nl)*

**Pull request merged**